automated/linux/mmtests/mmtests.sh

```diff
@@ -18,191 +15,199 @@ MMTESTS_MAX_RETRIES=10
 MMTESTS_TYPE_NAME=
 MMTESTS_CONFIG_FILE=
 
+# DBENCH specific variables
 declare -A altreport_mappings=( ["dbench4"]="tput latency opslatency")
 declare -A env_variable_mappings=( ["dbench4"]="DBENCH" )
 
 usage() {
-	echo "\
-	Usage: $0 [-s <true|false>] [-v <TEST_PROG_VERSION>]
-			[-u <TEST_GIT_URL>] [-p <TEST_DIR>]
-			[-c <MMTESTS_CONFIG_FILE>] [-t <MMTESTS_TYPE_NAME>] [-r <MMTESTS_MAX_RETRIES>]
-
-	<TEST_PROG_VERSION>:
-	If this parameter is set, then the ${TEST_PROGRAM} suite is cloned. In
-	particular, the version of the suite is set to the commit
-	pointed to by the parameter. A simple choice for the value of
-	the parameter is, e.g., HEAD. If, instead, the parameter is
-	not set, then the suite present in TEST_DIR is used.
-
-	<TEST_GIT_URL>:
-	If this parameter is set, then the ${TEST_PROGRAM} suite is cloned
-	from the URL in TEST_GIT_URL. Otherwise it is cloned from the
-	standard repository for the suite. Note that cloning is done
-	only if TEST_PROG_VERSION is not empty
-
-	<TEST_DIR>:
-	If this parameter is set, then the ${TEST_PROGRAM} suite is cloned to or
-	looked for in TEST_DIR. Otherwise it is cloned to $(pwd)/${TEST_PROGRAM}
-
-	<SKIP_INSTALL>:
-	If you already have it installed into the rootfs.
-	default: false
-
-	<MMTESTS_CONFIG_FILE>:
-	Mmtests configuration file that describes how the benchmarks should be
-	configured and executed.
-
-	<MMTESTS_TYPE_NAME>:
-	Mtests test type, e.g. sysbenchcpu, iozone, sqlite, etc.
-
-	<MMTESTS_MAX_RETRIES>:
-	Maximum number of retries for the single benchamrk's source file download"
-	exit 1
+  echo "\
```

Comment:
Overall comment, I'm not sure why we reformat the the file from tabs to spaces.

Reply:
1. Consistency - no mix tabs / spaces.
2. Spaces took less space.